packages/mui-material/src/IconButton/iconButtonClasses.ts

```diff
@@ -13,6 +13,14 @@ export interface IconButtonClasses {
   colorPrimary: string;
   /** Styles applied to the root element if `color="secondary"`. */
   colorSecondary: string;
+  /** Styles applied to the root element if `color="error"`. */
+  colorError: string;
+  /** Styles applied to the root element if `color="info"`. */
+  colorInfo: string;
+  /** Styles applied to the root element if `color="success"`. */
+  colorSuccess: string;
+  /** Styles applied to the root element if `color="warning"`. */
+  colorWarning: string;
```

Comment:
Please add unit tests for these color classes, Thanks!

Reply:
I added some unit tests. How do they look?

Reply:
LGTM!